tasks/build/package_scripts/pre_remove.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+echo -n "Stopping kibana service..."
+if command -v systemctl >/dev/null; then
+    systemctl --no-reload stop kibana.service
+fi
+if [ -x /etc/init.d/kibana ]; then
+    if command -v invoke-rc.d >/dev/null; then
+        invoke-rc.d kibana stop
+    else
+        /etc/init.d/kibana stop
```

Comment:
Do we want to try `service kibana stop` first, before trying `/etc/init.d/kibana stop`? Asking because it seems using `service` is now preferred to using `/etc/init.d` directly:

* http://serverfault.com/questions/85621/to-restart-a-service-e-g-httpd-should-i-use-etc-init-d-httpd-restart-or-sbi
* http://stackoverflow.com/questions/24836715/difference-between-etc-init-d-and-sbin-service
* http://stackoverflow.com/questions/22509654/difference-between-starting-a-command-using-init-d-script-and-service-start